include/zephyr/bindesc.h

```diff
@@ -48,6 +48,9 @@ extern "C" {
 /** The app version number such as 0x10203 */
 #define BINDESC_ID_APP_VERSION_NUMBER 0x804
 
+/** The app git reference such as "v3.3.0-18-g2c85d9224fca" */
+#define BINDESC_ID_APP_BUILD_VERSION 0x805
```

Comment:
please read this comment, and update the other file accordingly:
https://github.com/zephyrproject-rtos/zephyr/blob/77f02bef8259f548e937f01e09aedd13297bbc63/include/zephyr/bindesc.h#L32-L33

Reply:
Thanks for pointing that one out... resolved.